applications/dashboard/settings/class.hooks.php

```diff
@@ -214,7 +214,7 @@ public function dashboardNavModule_init_handler($sender) {
             ->addGroupToSection('Moderation', t('Content'), 'moderation')
             ->addLinkToSectionIf($session->checkPermission(['Garden.Moderation.Manage', 'Moderation.Spam.Manage'], false), 'Moderation', t('Spam Queue'), '/dashboard/log/spam', 'moderation.spam-queue', '', $sort)
             ->addLinkToSectionIf($session->checkPermission(['Garden.Moderation.Manage', 'Moderation.ModerationQueue.Manage'], false), 'Moderation', t('Moderation Queue'), '/dashboard/log/moderation', 'moderation.moderation-queue', '', $sort, ['popinRel' => '/dashboard/log/count/moderate'], false)
-            ->addLinkToSectionIf('Garden.Settings.Manage', 'Moderation', t('Change Log'), '/dashboard/log/edits', 'moderation.change-log', '', $sort)
+            ->addLinkToSectionIf($session->checkPermission(['Garden.Settings.Manage', 'Garden.Moderation.Manage']), 'Moderation', t('Change Log'), '/dashboard/log/edits', 'moderation.change-log', '', $sort)
```

Comment:
I think you want this to be:

```php
$session->checkPermission(['Garden.Settings.Manage', 'Garden.Moderation.Manage'], false)
```

Unless you include `false` as the second parameter, it's going to require they have Garden.Settings.Manage *and* Garden.Moderation.Manage.